packages/react-devtools-shared/src/devtools/views/Profiler/CommitRankedListItem.js

```diff
@@ -38,6 +46,23 @@ function CommitRankedListItem({data, index, style}: Props) {
     [node, selectFiber],
   );
 
+  const handleMouseOver = useCallback(
+    (event: SyntheticMouseEvent<*>, nodeData: ChartNodeType) => {
+      const {id, name} = nodeData;
+      event.stopPropagation();
+      hoverFiber({id, name});
+    },
+    [hoverFiber],
+  );
```

Comment:
As`nodeData` can be accessed from closure, maybe
```js
  const handleMouseOver = useCallback(
    (event: SyntheticMouseEvent<*>) => {
      const {id, name} = node;
      event.stopPropagation();
      hoverFiber({id, name});
    },
    [hoverFiber, node],
  );
```

Reply:
Makes sense, I removed it as param and used the value from closure with a simple callback